Kickstarter-iOS/Views/Cells/BackingCell.swift

```diff
@@ -4,14 +4,30 @@ import Prelude
 import ReactiveExtensions
 import UIKit
 
+internal protocol BackingCellDelegate: class {
+  /// Call when should navigate to Backing Info.
+  func backingCellGoToBackingInfo()
+}
+
```

Comment:
Previously, the "Backing info" button was hooked up directly as an `IBOutlet` to `MessagesViewController` so I made it a proper delegate.